### ISSUE TITLE
Suggestion: Enable quoted posts

### ISSUE THREAD

**Issue by forenta** [magicstone-dev/ecko#477]
Contrary to the interest of the Mastodon community, Eugen's denies quoted posts until now. Because of this, [a plug-in for this feature was developed by ariadne](https://gitea.treehouse.systems/treehouse/mastodon/pulls/36).
At this point, I would like to suggest that this plug-in becomes part of Ecko.

**Comment by politas** [comment]
What problem does making this plugin part of Ecko solve, given the widely discussed harassment issue?

**Comment by forenta** [comment]
To stop the problem, it is inevitable to prevent the use of screenshots in Ecko.

The causal relationship between harassment and quoted posts are an unsubstantiated legend of the Czar of Mastodon.

**Comment by jorgesat** [comment]
If the quoted user got a notification, they would be able to report any harassment situation. That would be a way to approach the problem.

**Comment by politas** [comment]
It's not just Eugen's dictate, and the issue is not one that has only come up with regard to Mastodon. I'm not aware of any polling on the Mastodon community's interests with regard to this point. Do you have some data to support your assertion?

Here is a good article outlining the fundamental ideas against "reblogging with addition" or quote-boost: https://www.pillowfort.social/posts/312460

One of the best points I see is that the quote-boost when used to disagree with a post has the effect of increasing the circulation of the post you are disagreeing with, which trends towards more disagreement.

In any case, Ecko operates on the C4 principles. Your issue needs to be stated as a problem. You haven't stated a problem that needs to be solved.

**Comment by politas** [comment]
> If the quoted user got a notification, they would be able to report any harassment situation. That would be a way to approach the problem.

This is not effective in the circumstance of pile-on harassment attacks, and it puts most of the effort of dealing with harassment on the victim.

**Comment by politas** [comment]
Another interesting article on the issue (neither negative nor positive) is here:  https://absolutelymaybe.plos.org/2022/12/01/reflecting-on-twitter-white-flight-quote-tweet-tensions-at-mastodon/

**Comment by weex** [comment]
It's true, we don't do Suggestions and I have a hard time coming up with the problem statement that I agree with. Thanks @politas for the posts above which I hope to review before commenting further.

**Comment by forenta** [comment]
> Another interesting article on the issue (neither negative nor positive) is here

Rather linked within the post here: https://absolutelymaybe.plos.org/2023/01/12/quote-tweeting-over-30-studies-dispel-some-myths/

>     Do QTs increase or decrease the spread of misinformation?
>         There wasn’t much evidence on this question. QTs weren’t a major vector of misinformation in these studies, and in one, debunking QTs spread faster than false stories.

>     Do QTs increase incivility and/or abuse?
>         The primary vectors of incivility and abuse on Twitter appear to be replies and hashtags, and it comes from a small minority of users.
>         There’s conflicting evidence on whether QTs increase or decrease incivility, and whatever effect there is, it doesn’t seem to be major.

> It's true, we don't do Suggestions

ok

**Comment by forenta** [comment]
> I'm not aware of any polling on the Mastodon community's interests with regard to this point. Do you have some data to support your assertion?

6.7k votes, 81% voted yes: https://tapbots.social/@paul/109761937953514834
Following the vote, the Czar has announced the feature on Mastodon-side, so there's no need anymore to do this here or elsehwere, I think.
![image](https://user-images.githubusercontent.com/2451238/215287907-fed949c6-206f-4dc2-be18-37e674004173.png)

**Comment by politas** [comment]
Might want to re-open this, since it hasn't been implemented in Ecko.

**Comment by weex** [comment]
Right, we haven't decided whether this is a Problem or not. I'll change the title as well.
